docs/dashboards/light-dark-theme.md

```diff
@@ -0,0 +1,16 @@
+Light and Dark Theme
+===
+
+The Light and Dark theme allows you to switch between a light and dark theme for the dashboard.  
+To use the Light and Dark theme, you need to import the `gui.css` file.
+
+```css
+    @import "https://code.highcharts.com/css/dashboards/gui.css";
+```
+
+By default, the theme is set to light. But it adjusts itself to the user's system settings.  
```

Comment:
More precisely, it defaults to the system default through the `prefers-color-scheme` media query.